bucket/danser-go.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.6.7",
+  "description": "Dancing visualizer of osu! standard maps and custom osu! client written in Go. Also a generator for osu! videos.",
+  "homepage": "https://github.com/Wieku/danser-go",
+  "license": "MIT",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/Wieku/danser-go/releases/download/0.6.8/danser-0.6.8-win.zip#dl.7z",
+      "hash": "b54b43df55f09a239e55078759e8c84ed085d8dd4a436f34fa6faae5db095580"
+    }
+  },
+  "bin": "danser.exe",
+  "persist": "settings",
```

Comment:
Is this a file or a folder?

Reply:
> Is this a file or a folder?

settings is a folder created when first executing danser which holds all the json configuration files